### DISCUSSION TITLE
Misaligned Title while adding map

### DISCUSSION THREAD

**Discussion by JonathanGLopez** [albertkun/23S-ASIAAM-191A#15] [Help Desk]
For whatever reason my heading shifted to the bottom when I added the map while completing last week's in-class assignment. Any suggestions on why this may be? http://127.0.0.1:5500/Week2/index.html
@albertkucla 
![Screenshot 2023-04-25 at 15-25-19 Hello World](https://user-images.githubusercontent.com/121324059/234417830-662eb117-8d9d-4b8b-9345-d176e6390b97.png)

**Comment by JonathanGLopez** [comment]
@albertkun

**Reply by albertkun** [reply]
@JonathanGLopez could you commit the latest code to your repository? It's probably an issue related to your html tags.
